PokeDemo/Screens/PokemonList/Model/Pokemon.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Pokemon.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import Foundation
+
+struct Pokemon: Decodable {
+    var name: String
+    var url: String
+    
+    enum CodingKeys: String, CodingKey {
+        case name
+        case url
+    }
```

Comment:
Do we need CodingKeys?

Reply:
Covered in a4731a787e76e1973e94509f7945ebccdc9fc4e1